docs/dev/building-on-zksync/hello-world.md

```diff
@@ -1,80 +1,66 @@
 # Quickstart
 
-In this quickstart guide, you will learn how to deploy a smart contract to zkSync and build a dApp to interact with it using the zkSync development toolbox.
-
-This is what we're going to build:
-
-- A smart contract that stores a greeting message and is deployed on zkSync.
-- A dApp to retrieve and update the greeting message.
-- Users will be able to change the greeting on the smart contract.
-- By default, users will pay transaction fees in ETH. However, we will also explain how to [implement the testnet paymaster](#paying-fees-using-testnet-paymaster) to allow users to pay transaction fees with ERC20 tokens.
-
+This guide shows you how to deploy a smart contract to zkSync and build a dApp that interacts with it using the zkSync development toolbox.
 
+This is what we're going to do:
 
+- Build and deploy a smart contract on zkSync that stores a greeting message.
+- Build a dApp that retrieves and updates the greeting message.
+- Allow users to change the greeting message on the smart contract via the app.
+- Show you how to [implement the testnet paymaster](#paying-fees-using-testnet-paymaster) that allows users to pay transaction fees with ERC20 tokens instead of ETH.
 
 ::: warning
-
-Please note that breaking changes were introduced in `zksync-web3 ^0.13.0`. The API layer now operates with `gas` and the `ergs` concept is only used internally by the VM.
-
+- Breaking changes were introduced in `zksync-web3 ^0.13.0`. 
+- The API layer now operates with `gas` and the `ergs` concept is only used internally by the VM.
 :::
 
 ## Prerequisites
 
-- `yarn` or `NPM` package manager. We recommend `yarn` and it's what we used in the front-end project. [Here is the Yarn installation guide](https://yarnpkg.com/getting-started/install) in case you don't have it.
-- A wallet with sufficient Göerli `ETH` on L1 to pay for bridging funds to zkSync as well as deploying smart contracts.
-- ERC20 tokens on zkSync are required if you want to implement the testnet paymaster. We recommend using [the faucet from the zkSync portal](https://portal.zksync.io/faucet).
+- Download and install [Node](https://nodejs.org/en/download).
+- Use the `yarn` or `npm` package manager. We recommend using `yarn`. To install `yarn`, follow the [Yarn installation guide](https://yarnpkg.com/getting-started/install).
+- A wallet with sufficient Göerli `ETH` on L1 to pay for bridging funds to zkSync and deploying smart contracts.
+- ERC20 tokens on zkSync are required for the testnet paymaster. We recommend using [the faucet from the zkSync portal](https://portal.zksync.io/faucet).
+- You know [how to get your private key from your MetaMask wallet](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
 
-## Initializing the project & deploying a smart contract
+:::info
+This tutorial was recently tested running Node vv16.16.0.
+:::
```

Comment:
Shouldn't this be in the prerequisite section, like "We recommend using Node v16.16.0 for this tutorial", instead of telling the reader what Node version was used to test this tutorial? @kailash-manasarovar

Reply:
```suggestion
:::info
This tutorial was recently tested running Node vv16.16.0.
:::
```

BTW, there are double V's, ->  "Node v16.16.0".